code/modules/mining/machine_vending.dm

```diff
@@ -184,6 +184,8 @@
 		if("Resonator Kit")
 			new /obj/item/extinguisher/mini(drop_location)
 			new /obj/item/resonator(drop_location)
+		if("Neural Lace Kit") // Skyrat edit
+			new /obj/item/autosurgeon/organ/corticalstack(drop_location)
```

Comment:
It'd be fantastic if you could buy them through the vendor for points and stuff too

Reply:
Soon(TM)

Reply:
Thanks